src/test/java/org/commcare/formplayer/tests/FilterTests.java

```diff
@@ -34,4 +34,14 @@ public void testSyncDb() throws Exception {
 
         //TODO add ledgers, fixtures, etc.
     }
+
+    @Test
+    public void testIntervalSyncDb() throws Exception {
```

Comment:
If you're going to put this in this test class, I'd suggest renaming it to `SyncTests` or something more appropriate.

It would also be great if you migrated it to the new style of tests. Alternately create a new test file in the new style and leave this one as is.

Reply:
migrated [eb7fa3a](https://github.com/dimagi/formplayer/pull/1544/commits/eb7fa3ae36a8ebb9bea04cd3a5df4557128cca44)

Renamed class name [50554fa](https://github.com/dimagi/formplayer/pull/1544/commits/50554faa9f597a988ad56e71dfa42e12ff44d1ec)